AssociateAPI/src/main/java/com/bae/associate/Associate.java

```diff
@@ -0,0 +1,12 @@
+package com.bae.associate;
```

Comment:
Just to check, should this be in a sub-package? Or should be at the same level as main?